src/Components/SmartComponents/CVEs/CVEs.js

```diff
@@ -93,7 +93,7 @@ export const CVEs = ({ rbac }) => {
         };
 
         apply(filtersOnLoad);
-    }, [shouldUseHybridSystemFilter]);
+    }, [shouldUseHybridSystemFilter. urlParameters]);
```

Comment:
Why urlParameters are needed here in the dependencies list? This useEffect deals with the first load and it seems to me that we should leave it empty completely, or am I wrong?